src/pilot/templates/create_aggregate_environment.yaml

```diff
@@ -0,0 +1,4 @@
+parameters:
+  aggregate_name: HugePage_Aggr
+  aggregate_metadata: {'hugepages': 'True'}
+  hosts: [director.r146.nfv.lab]
```

Comment:
dont think you want the dns hardcoded there

Reply:
This is not hard coded value
Following line will replace this with real time value
dell_nfv.py :
cmds.append('sed -i "s|hosts:.*|hosts: ' + str(hostname_list) + '|" ' + file_path)

Reply:
What i meant is you should delete "director.r146.nfv.lab" .

Reply:
Ok I will update the template as follows 

parameters:
  aggregate_name:
  aggregate_metadata:
  hosts:

Reply:
I think only one aggregate is fine. Besides, we don't need multiple aggregate metadata, we can for example set a metadata to csp=true or whatever name can be can be suitable. We, then, define flavors with appropriate metadata that will target this metadata and we're all set. NUMATopology filter will choose which node to pick up to do the right placement for the VM based on the need of the flavor (hugepages number and size, and/or numa).